extensions/clusters/kubernetesversions/all.go

```diff
@@ -215,23 +215,19 @@ func sortReleases(releases []interface{}) (allAvailableVersions []string) {
 			continue
 		}
 
-		minVersion := release.(map[string]interface{})["minChannelServerVersion"].(string)
 		kubernetesVersion := release.(map[string]interface{})["version"].(string)
 
-		minRancherVersion, err := semver.NewVersion(strings.TrimPrefix(minVersion, "v"))
-		if err != nil {
-			continue
-		}
-
 		releaseKubernetesVersion, err := semver.NewVersion(strings.TrimPrefix(kubernetesVersion, "v"))
 		if err != nil {
 			continue
 		}
 
-		value, ok := availableVersionsMap[minRancherVersion.String()]
+		minor := strings.Split(releaseKubernetesVersion.Original(), ".")[1]
```

Comment:
qq, what does releaseKubernetesVersion.Original() return here? can we not just use releaseKubernetesVersion?

Reply:
.Original returns the string of releaseKuberenetesVersion, which is of type semver.Version. So we need the conversion to string here